draft-demarco-oauth-status-assertions.md

```diff
@@ -676,6 +676,125 @@ may not always coincide with the actual usability of a Digital Credential,
 allowing Verifiers to examine and make educated conclusions based on a
 variety of scenarios.
 
+# Complex Status Assertions
+
+Status assertions can be complex, and are not limited to simple boolean information.
+This enables verifier policies to be conditioned on the presence of secured information, instead of the absence of information.
+This section proposes syntax to support complex assertions.
+The `credential_status_validity` claim MUST be present and be either `true` or `false`.
+The `credential_status` claim MAY be present and if present MUST be an object.
+The semantics of the claims within the `credential_status` object are determined by the issuer.
+
+An example of a boolean status is:
+
+~~~
+{
+    "alg": "ES256",
+    "typ": "status-assertion+jwt",
+    "kid": "w8ZOZRcx21Zpry7H-0VLBsH7Wf7WXb6TeK3qVMCpY44"
+}
+.
+{
+    "iss": "https://issuer.example.org",
+    "iat": 1504699136,
+    "exp": 1504785536,
+    "credential_hash": "xnlAq6Ma8fgu1z4hdGphJnKLulaVHpLCFeZFUGpQ2dA",
+    "credential_hash_alg": "sha-256",
+    "credential_status_validity": false,
+    "credential_status": {
+      "suspended": true,
```

Comment:
What about define at least two known credential_statuses

suspendend, revoked

(we might find them also in the EUDIW ARF 1.4)